app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -49,14 +48,13 @@
             android:title="@string/preferences_sync_logins" />
 
         <CheckBoxPreference
-            android:defaultValue="false"
+            android:defaultValue="true"
             android:key="@string/pref_key_sync_tabs"
             android:layout="@layout/checkbox_left_preference"
             android:title="@string/preferences_sync_tabs_2"/>
 
         <CheckBoxPreference
-            android:defaultValue="false"
-            android:visible="false"
```

Comment:
I think we should avoid removing the default visibility hidden here. Otherwise, we would always see the preference showing and being hidden in the release build.

Reply:
Ah, good call.